src/MultiTypeSymEntry.chpl

```diff
@@ -110,6 +114,9 @@ module MultiTypeSymEntry
             if v {writeln("deinit SymEntry");try! stdout.flush();}
         }
         
+        override proc writeThis(f) throws {
+          halt("SymEntry cannot be stringified");
+        }
```

Comment:
These halts helped me find a few other cases. 

If you want to avoid halts, we could make this a warning (though I worry we won't see the warning), or we could just define how to print symbol entries in a way that doesn't print the whole array.

Reply:
@ronawho First of all, excellent work on figuring this out. I see that I changed a few st.lookup to st.attrib, just didn't catch 'em all. Since this one returns a string, it should be good to replace the name with st.attrib. 

From my grepping about the only one left is InqdMsg.chpl

Reply:
Swapped out the halts for printing things, but limiting how much of the array is printed (this was used as part of `SymTab.pretty()`, which is used in a few tests. 

I also updated the instance in In1dMsg.chpl